README.fr.md

```diff
@@ -45,11 +45,16 @@ chmod +x ./env_variables.sh
 
 ## Démarrer le code
 
-- Pour générer les classes pour l'internationalisation, exécuter les commandes suivantes:
-```bash
+- Pour générer les classes pour l'internationalisation, exécuter la commande suivante:
+```sh
 flutter pub get
 ```
 
+- Une fois les dépendances correctement importées, il est possible de lancer l'application en mode dev en exécutant la commande suivante:
+```sh
+$ flutter run --debug -t lib/main_dev.dart --flavor=dev
```

Comment:
Is it possible to have dev been the default flavor?

Reply:
I don't think it's possible to set default flavor from flutter, but I could rename the main_dev.dart file to main.dart (so it's launch as default) and make the default dev flavor in Android. I'm not sure I'm able to do that on iOS will have to look that!

If I can't, It still possible to setup launch settings from intellij or vscode 😄 In that case, I'll add a README entry to setup that.

Reply:
With further investigation it is not currently possible to make a default flavor to build :( https://github.com/flutter/flutter/issues/22856 I will still be able to rename main_dev.dart to main.dart. so at least the only argument to pass now is `--flavor=dev`. I also updated the README.md documentation.